Sources/AppcuesKit/Presentation/ExperienceRendering/ExperienceRenderer.swift

```diff
@@ -92,7 +97,7 @@ internal class ExperienceRenderer: ExperienceRendering, StateMachineOwning {
         }
 
         guard let stateMachine = stateMachines[experience.renderContext] else {
-            potentiallyRenderableExperiences[experience.renderContext] = [experience]
+            analyticsObserver.trackRecoverableError(experience: experience, message: "no render context \(experience.renderContext)")
```

Comment:
Checking understanding - so the addition to cache is not needed here any longer since that is handled in the new L86. Pretty sure that is correct 👍

Reply:
Yep, exactly. This was problematic for the reasons described in pt 1 of the description.